packages/monitor-v2/src/monitor-og/oSnapAutomation.ts

```diff
@@ -213,28 +211,35 @@ const getUndiscardedProposals = async (
       })
     )
   ).flat();
+};
 
-  if (params.reproposeDisputed) {
-    // Get all deleted proposals for all provided modules.
-    const deletedProposals = (
-      await Promise.all(
-        ogAddresses.map(async (ogAddress) => {
-          const og = await getOgByAddress(params, ogAddress);
-          return runQueryFilter<ProposalDeletedEvent>(og, og.filters.ProposalDeleted(), {
-            start: 0,
-            end: params.blockRange.end,
-          });
-        })
-      )
-    ).flat();
+// Filters all proposals that have not been discarded. Discards are most likely due to disputes, but can also occur on
+// OOv3 upgrades.
+const filterUndiscardedProposals = async (
```

Comment:
Should we call this `removeDeletedProposals`? I find the current name a bit confusing.

Reply:
makes sense, pushed the fix for rename